bin/semver

```diff
@@ -56,6 +75,7 @@ function main () {
         break
       case "-h": case "--help": case "-?":
         return help()
+        break
```

Comment:
I guess the break here was missing. This is not the main purpose of the patch though.